features/registerHealthCertificate.feature

```diff
@@ -0,0 +1,29 @@
+@RegistryService
+Feature: Register Health Certificate
+
+    @RegisterHealthCertificate
+    Scenario: Submit Valid Register Health Certificate Transaction
+        Given I make a POST request to /
+            And I set json body to the file at ./features/fixtures/registerHealthCertificate/Bundle-ProvideDDCCDocument.json
+        When I receive a response
+        Then I expect response should have a status 200
+            And I expect response should validate against the profile http://profiles.ihe.net/ITI/MHD/StructureDefinition/IHE.MHD.ProvideDocumentBundleResponse
+
+    @RegisterInvalidHealthCertificate-400
```

Comment:
Should we have tags for each scenario? I added them in case we need them for the future, but I am also open to getting rid of them or consolidating them

Reply:
I don’t think it hurts, and like you said, it provides a way to refer to them in the future if needed.